src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/help-cloudStatusIntervalSec.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Set the interval for checking the status of the EC2 cloud.
```

Comment:
it will be good to mention default value and when we actually update interval.

Reply:
ok